packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -525,7 +525,6 @@ class _GestureSemantics extends SingleChildRenderObjectWidget {
         return;
       }
     }
-    assert(false);
```

Comment:
These lines are hittable.  The OS can ask you to perform an action that you're no longer interested in because the whole ax system is async.